src/components/apphost.js

```diff
@@ -1,447 +1,412 @@
-define(['appSettings', 'browser', 'events', 'htmlMediaHelper', 'webSettings', 'globalize'], function (appSettings, browser, events, htmlMediaHelper, webSettings, globalize) {
-    'use strict';
+import appSettings from 'appSettings';
+import browser from 'browser';
+import events from 'events';
+import * as htmlMediaHelper from 'htmlMediaHelper';
+import * as webSettings from 'webSettings';
+import globalize from 'globalize';
 
-    browser = browser.default || browser;
+function getBaseProfileOptions(item) {
+    const disableHlsVideoAudioCodecs = [];
 
-    function getBaseProfileOptions(item) {
-        var disableHlsVideoAudioCodecs = [];
-
-        if (item && htmlMediaHelper.enableHlsJsPlayer(item.RunTimeTicks, item.MediaType)) {
-            if (browser.edge) {
-                disableHlsVideoAudioCodecs.push('mp3');
-            }
-
-            disableHlsVideoAudioCodecs.push('ac3');
-            disableHlsVideoAudioCodecs.push('eac3');
-            disableHlsVideoAudioCodecs.push('opus');
+    if (item && htmlMediaHelper.enableHlsJsPlayer(item.RunTimeTicks, item.MediaType)) {
+        if (browser.edge) {
+            disableHlsVideoAudioCodecs.push('mp3');
         }
 
-        return {
-            enableMkvProgressive: false,
-            disableHlsVideoAudioCodecs: disableHlsVideoAudioCodecs
-        };
-    }
-
-    function getDeviceProfileForWindowsUwp(item) {
-        return new Promise(function (resolve, reject) {
-            require(['browserdeviceprofile', 'environments/windows-uwp/mediacaps'], function (profileBuilder, uwpMediaCaps) {
-                var profileOptions = getBaseProfileOptions(item);
-                profileOptions.supportsDts = uwpMediaCaps.supportsDTS();
-                profileOptions.supportsTrueHd = uwpMediaCaps.supportsDolby();
-                profileOptions.audioChannels = uwpMediaCaps.getAudioChannels();
-                resolve(profileBuilder(profileOptions));
-            });
-        });
+        disableHlsVideoAudioCodecs.push('ac3');
+        disableHlsVideoAudioCodecs.push('eac3');
+        disableHlsVideoAudioCodecs.push('opus');
     }
 
-    function getDeviceProfile(item, options) {
-        options = options || {};
-
-        if (self.Windows) {
-            return getDeviceProfileForWindowsUwp(item);
-        }
+    return {
+        enableMkvProgressive: false,
+        disableHlsVideoAudioCodecs: disableHlsVideoAudioCodecs
+    };
+}
 
-        return new Promise(function (resolve) {
-            require(['browserdeviceprofile'], function (profileBuilder) {
-                var profile;
+function getDeviceProfile(item, options = {}) {
+    return new Promise(function (resolve) {
+        import(['browserdeviceprofile']).then(({default: profileBuilder}) => {
```

Comment:
```suggestion
        import('browserdeviceprofile').then(({default: profileBuilder}) => {
```
Same here https://github.com/jellyfin/jellyfin-web/blob/c0ac2f5fc3c0aca78c0cba34596dcba06b003e4a/src/controllers/livetv/livetvchannels.js#L94
But it seems to work.